pkg/features/kube_features.go

```diff
@@ -227,6 +227,7 @@ var defaultVersionedFeatureGates = map[featuregate.Feature]featuregate.Versioned
 	},
 	LocalQueueMetrics: {
 		{Version: version.MustParse("0.10"), Default: false, PreRelease: featuregate.Alpha},
+		{Version: version.MustParse("0.11"), Default: true, PreRelease: featuregate.Beta},
```

Comment:
Is there a reason why it needs to be bundled? If not, please make it a separate PR